docs/en/datasets/det.md

```diff
@@ -63,6 +63,9 @@ The structure of the text detection dataset directory is organized as follows.
 | Lecture Video DB  |                                                                                               [homepage](https://cvit.iiit.ac.in/research/projects/cvit-projects/lecturevideodb)                                                                                               |                                                                                                              -                                                                                                               |                                              -                                               |                                               -                                                |
+|     IIIT-ILST     |                                                                                                  [homepage](http://cvit.iiit.ac.in/research/projects/cvit-projects/iiit-ilst)                                                                                                  |                                                                                                              -                                                                                                               |                                              -                                               |                                               -                                                |
+|      VinText      |                                                                                                            [homepage](https://github.com/VinAIResearch/dict-guided)                                                                                                            |                                                                                                              -                                                                                                               |                                              -                                               |                                               -                                                |
+|        BID        |                                                                                               [homepage](https://drive.google.com/file/d/1Oi88TRcpdjZmJ79WDLb9qFlBNG8q2De6/view)                                                                                               |                                                                                                              -                                                                                                               |                                              -                                               |                                               -                                                |
```

Comment:
It might be better to link the homepage to the Github website https://github.com/ricardobnjunior/Brazilian-Identity-Document-Dataset